src/relay/backend/contrib/cmsisnn/scalar_to_tensor_constant.cc

```diff
@@ -86,63 +85,62 @@ class ScalarToTensorConstantMutator : public MixedModeMutator {
       final_call = Call(global_var, call->args);
     }
 
-    // Substitute scalar constant with a tensor constant in the call to composite function
-    // comprising partitioned binary ops. Shape of the new constant should be same as its
-    // neighbouring tensor's shape.
+    // Substitute scalar constant with tensor constant in the call to composite function.
     if (auto* func_node = call->op.as<FunctionNode>()) {
       Function func = GetRef<Function>(func_node);
-      auto func_name = func->GetAttr<String>(attr::kComposite);
-      if (func_name.defined() &&
-          (func_name == "cmsis-nn.qnn_add" || func_name == "cmsis-nn.qnn_mul")) {
-        final_call = ReplaceScalarWithTensorConstant(GetRef<Call>(call), func);
-      }
+      final_call = ReplaceScalarWithTensorConstant(GetRef<Call>(call), func);
     }
 
     return final_call;
   }
 
   // Replaces scalar variable with a tensor variable with same shape as that of the neibouring
```

Comment:
[Unrelated to this PR, feel free ignore] : a typo : neibouring

Reply:
ACK